string costmap_name
ObstructionMsg[] obstructions